spec/system/batch_process_spec.rb

```diff
@@ -62,15 +53,9 @@
       end
 
       it "can still see the details of the import" do
-        within "td.process_id" do
-          expect(page).to have_link(BatchProcess.last.id.to_s, href: "/batch_processes/#{BatchProcess.last.id}")
-        end
-        within "td.count" do
-          expect(page).to have_content('5')
-        end
-        within "td.view" do
-          expect(page).to have_link('View', href: "/batch_processes/#{BatchProcess.last.id}")
-        end
+        expect(page).to have_link(BatchProcess.last.id.to_s, href: "/batch_processes/#{BatchProcess.last.id}")
+        expect(page).to have_content('5')
+        expect(page).to have_link('View', href: "/batch_processes/#{BatchProcess.last.id}")
```

Comment:
I tried something similar to this in another test above but it didn't work for me. maybe the difference was adding `js:true`?